### PR TITLE
Change from Apps to "More Apps"

### PR DESCRIPTION
**Pull request by timlevett**
image

**What is this feature?**

This is a label change for the "other" bucket of apps in the menu.

**Why do we need this feature?**

Right now in Grafana Cloud there is both "Apps" and "Applications", by prefixing Apps with More it signifies that this is an other bucket, and not application observability.

**Who is this feature for?**

Detail oriented users.

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/contribute/release-notes/#how-to-determine-if-content-belongs-in-whats-new), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/contribute/release-notes/) doc.

### CODE REVIEW

File: pkg/services/navtree/navtreeimpl/applinks.go [hunk 1]
```diff
@@ -195,7 +195,7 @@ func (s *ServiceImpl) addPluginToSection(c *contextmodel.ReqContext, treeRoot *n
 		switch sectionID {
 		case navtree.NavIDApps:
 			treeRoot.AddSection(&navtree.NavLink{
-				Text:       "Apps",
+				Text:       "More Apps",
```
Comment (ashharrison90) [inline pkg/services/navtree/navtreeimpl/applinks.go:198]: this should be sentence case: `More apps`

**Review by ashharrison90 — CHANGES_REQUESTED** [review]
you don't need to directly edit the translation files. see the internationalisation documentation [here](https://github.com/grafana/grafana/blob/main/contribute/internationalization.md). instead:
- edit the translation [here](https://github.com/grafana/grafana/blob/timlevett/more-apps/public/app/core/utils/navBarItem-translations.ts#L141)
- run `make i18n-extract`

edit: forgot to add, i don't really have a strong opinion on the naming either way 🤷

**Review by joshhunt — COMMENTED** [review]
pushed up:
 - changed case to `More apps`
 - removed manual changes from translations (that happens in a workflow outside of the PR)
 - ran i18n-extract to get everything all good :)

**Review by ashharrison90 — APPROVED** [review]
lgtm 👍

**Comment by uxdmitryn** [comment]
LGTM 👍

**Pull request merged**